src/Components/Authorization/test/AuthorizeViewTest.cs

```diff
@@ -292,6 +292,68 @@ public void RendersNothingUntilAuthorizationCompleted()
             });
         }
 
+        [Fact]
+        public void RendersAuthorizingWhenAuthenticationIsCompletedBeforehandAndAuthorizationIsAsync()
```

Comment:
nit: Can we shorten this a bit, maybe RendersAuthorizingWhenAsync the current test name is just too long to easily grok

Reply:
Or RendersAuthorizingUntilAuthorizationCompletedAsync asume this is similar to the test below

Reply:
Sure

Reply:
Done